client/lib/analytics/utils/cost-to-usd.js

```diff
@@ -191,7 +191,7 @@ function isSupportedCurrency( currency ) {
  *
  * @param {number} cost - The cost of the cart or product
  * @param {string} currency - The currency such as `USD`, `JPY`, etc
- * @returns {string} Or null if the currency is not supported
+ * @returns {number} Or null if the currency is not supported
```

Comment:
I think this should include null in the type.

```suggestion
 * @returns {number|null} Or null if the currency is not supported
```

Reply:
That's a great catch indeed, thank you!!